### ISSUE TITLE
Is there a chance for drag and drop on iOS like android？

### ISSUE THREAD

**Issue by Okra0o0** [NetrisTV/ws-scrcpy#167]
As we all know, WDA provides HTTP requests for operating the phone , and if the user drags for a long time, the cloud phone will not respond in time. I also tried to send multiple requests for long drag-and-drop segments, but it triggered multiple click events, which didn't work as expected. So is there a chance for drag and drop on iOS like android？Thanks~

**Comment by drauggres** [comment]
Hi.
I don't think so, WDA was not really made to control devices in real time.

At the moment only three type of actions available from the client:
* simple click
* swipe (click - drag - release)
* home button press

It is possible to run an [action sequence](http://appium.io/docs/en/commands/interactions/touch/touch-perform/), but you must have a complete sequence before you send a request to WDA. On android we send simple events (pointer down, pointer move, pointer up), so it all works in real time.

P.S. To make the remote controll of an iOS device a little more productive, you can enable `AssistiveTouch` in the settings of your device.

**Comment by Okra0o0** [comment]
Yeah, you're absolutely  right. So I've been trying to control the remote device in real time without WDA, but  I haven’t gotten anywhere. Do you have any suggestions on this? Thanks~

**Comment by drauggres** [comment]
No. As far as I know, WDA is the only working solution for physical devices.

**Comment by Okra0o0** [comment]
Maybe. It's really a big problem!
